helix-core/src/test/java/org/apache/helix/common/ZkTestBase.java

```diff
@@ -710,7 +725,9 @@ protected Message createMessage(Message.MessageType type, String msgId, String f
   }
 
   @AfterClass
-  public void cleanupLiveInstanceOwners() {
+  public void cleanupLiveInstanceOwners() throws InterruptedException {
+    String testClassName = this.getShortClassName();
+    System.out.println("AfterClass:" + testClassName + " afterclass of ZkTestBase called!");
```

Comment:
Please remove this output. I think with no leakage found, we don't need this information.

Reply:
removed